### PR TITLE
feat: readiness return version info, sentry get git commit hash

### PR DESCRIPTION
**Pull request by terovirtanen**
[OH-56](https://helsinkisolutionoffice.atlassian.net/browse/OH-56)

[OH-56]: https://helsinkisolutionoffice.atlassian.net/browse/OH-56?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-API branch is deployed to platta: https://helsinki-profile-pr479.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-API branch is deployed to platta: https://helsinki-profile-pr479.api.dev.hel.ninja** :rocket::rocket::rocket:

File: open_city_profile/urls.py [hunk 1]
```diff
@@ -48,7 +50,13 @@ def healthz(*args, **kwargs):
...
 
 def readiness(*args, **kwargs):
-    return HttpResponse(status=200)
+    response_json = {
+        "status": "ok",
+        "packageVersion": __version__,
+        "commitHash": settings.COMMIT_HASH,
+        "buildTime": settings.APP_BUILD_TIME.strftime("%Y-%m-%dT%H:%M:%S.000Z"),
+    }
+    return HttpResponse(json.dumps(response_json), status=200)
```
Comment (charn) [inline open_city_profile/urls.py:59]: Django also has `JsonResponse` so you could do:
```suggestion
    return JsonResponse(response_json, status=200)
```

File: open_city_profile/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = "1.13.1"
```
Comment (charn) [inline open_city_profile/__init__.py:1]: I guess release-please automatically bumps this?

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-API branch is deployed to platta: https://helsinki-profile-pr479.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-API branch is deployed to platta: https://helsinki-profile-pr479.api.dev.hel.ninja** :rocket::rocket::rocket:

File: open_city_profile/__init__.py (hunk 2 above)
Comment (terovirtanen) [reply open_city_profile/__init__.py:1]: yes, release-please should bump this.

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-API branch is deployed to platta: https://helsinki-profile-pr479.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-API branch is deployed to platta: https://helsinki-profile-pr479.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Pull request merged**